src/Surfnet/Stepup/Identity/Event/RegistrationAuthorityInformationAmendedEvent.php

```diff
@@ -24,12 +24,21 @@
 use Surfnet\Stepup\Identity\Value\Institution;
 use Surfnet\Stepup\Identity\Value\Location;
 use Surfnet\Stepup\Identity\Value\NameId;
+use Surfnet\StepupMiddleware\CommandHandlingBundle\SensitiveData\RightToObtainDataInterface;
 
 /**
  * @deprecated This event is superseded by the RegistrationAuthorityInformationAmendedEvent because an RA institution was needed
  */
-class RegistrationAuthorityInformationAmendedEvent extends IdentityEvent
+class RegistrationAuthorityInformationAmendedEvent extends IdentityEvent implements RightToObtainDataInterface
 {
+    protected static $whitelist = [
+        'identity_id',
+        'name_id',
+        'institution',
+        'location',
+        'contact_information',
+    ];
```

Comment:
Sensitive data is missing

Reply:
there is no sensitive data handled in this class.